composer.json

```diff
@@ -11,9 +11,9 @@
     ],
     "require": {
         "php": ">=7.1",
-        "humanmade/wp-seo": "0.14.0",
         "humanmade/hm-redirects": "~0.5.1",
-        "humanmade/meta-tags": "^0.1.6"
+        "humanmade/meta-tags": "^0.1.6",
+        "yoast/wordpress-seo": "^16.2.0"
```

Comment:
It actually looks like they don't use minor point releases (`16.2.x`) very often at all, and their release cycle appears to be ~every two weeks (current version is 16.4, released yesterday, previous version, 16.3 was released 15 days ago). I'm not sure where that lands us in terms of following their schedule, but between Altis releases we're likely to get _very_ far behind on Yoast updates between one Altis release to the next. If I go back to the last Altis release (v6), the currently available version of Yoast SEO at that point was 15.7 with 15.8 in an RC4. They use the same basic versioning as WP (e.g. 16.0 comes after 15.9 and is not a significant release in itself), and their releases vary a lot in terms of what's being added. I'd say for the most part there isn't anything that's majorly breaking, but I don't think we'd get much value out of just updating the minor point releases since they just don't release those very often.

Anecdotally and unscientifically, it appears that roughly every other release there is more of an emphasis on features over minor enhancements and bugfixes, but there isn't anything that I'd identify as major changes in the last 5 releases that jumps out at me. For example, 16.4's major enhancements were around Czech language recognition features, whereas 16.3 added lots more Schema.org improvements.

Reply:
I see, it feels a bit odd but if there are unlikely to be breaking changes it could be ok. I'm still concerned that they could change some markup or something that seems minor but breaks our modifications meaning we'd have to patch it mid release cycle. Makes it hard for us to adapt proactively and could be a source of unscheduled or unexpected work.

Lets go with it as is for now though and re-evaluate closer to v8 depending on whether we've had to make any further updates.

Reply:
I agree that we should leave it for now, but I am thinking that maybe we lock Yoast releases to patch updates for Altis releases. Yoast may fall "behind" but I honestly don't know any professional services project where Yoast was up-to-date all the time, either.